Build/ProjectTypes/Analyzer.targets

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
We don't need the file unless there's actual logic in it (see Library.props). Please delete.